index.html

```diff
@@ -8,27 +8,64 @@
     <title>Portfolio</title>
   </head>
   <body>
-    <nav>
-      <div class="start"></div>
-      <div class="name">
-        <p>Henry Nawrocki</p>
-        <a href="#"><img src="./assets/icons/sandwich.png" alt="" /></a>
-      </div>
-      <div class="center"></div>
-      <ul>
-        <li><a href="#">Portfolio</a></li>
-        <li><a href="#">About</a></li>
-        <li><a href="#">Contact</a></li>
-        <li>
-          <a href="#"
-            ><img
-              class="email-icon"
-              src="./assets/icons/email.png"
-              alt="email icon"
-          /></a>
-        </li>
-      </ul>
-      <div class="end"></div>
-    </nav>
+    <header>
+      <nav>
+        <div class="start"></div>
+        <div class="name">
+          <p>Henry Nawrocki</p>
```

Comment:
- [ ] Kindly your logo should be inside `a` tag, so anyone who clicks on it switch him to the index page.

```
<a herf="index.html" class="welcome">Henry Nawrocki</a>
```